pages/common/ghcup.md

```diff
@@ -0,0 +1,33 @@
+# ghcup
+
+> Haskell toolchain installer.
+> Install, manage, and update Haskell toolchains.
+> More information: <https://gitlab.haskell.org/haskell/ghcup-hs>.
+
+- Start the interactive TUI:
+
+`ghcup tui`
+
+- List available GHC/cabal versions:
```

Comment:
It doesn't say "GHC/cabal version" below, so I think it's okay to leave it out here, too.
```suggestion
- List available GHC versions:
```

Reply:
This will list GHC/cabal versions, not just GHC. See example output from my machine:
```
✗  ghc   7.10.3         base-4.8.2.0
✗  ghc   8.0.2          base-4.9.1.0
✗  ghc   8.2.2          base-4.10.1.0
✗  ghc   8.4.1          base-4.11.0.0
✗  ghc   8.4.2          base-4.11.1.0
✗  ghc   8.4.3          base-4.11.1.0
✗  ghc   8.4.4          base-4.11.1.0
✗  ghc   8.6.1          base-4.12.0.0
✗  ghc   8.6.2          base-4.12.0.0
✗  ghc   8.6.3          base-4.12.0.0
✗  ghc   8.6.4          base-4.12.0.0             hls-powered
✗  ghc   8.6.5          base-4.12.0.0             hls-powered
✗  ghc   8.8.1          base-4.13.0.0
✗  ghc   8.8.2          base-4.13.0.0             hls-powered
✗  ghc   8.8.3          base-4.13.0.0             hls-powered
✔✔ ghc   8.8.4          recommended,base-4.13.0.0 hls-powered
✗  ghc   8.10.1         base-4.14.0.0             hls-powered
✗  ghc   8.10.2         base-4.14.1.0             hls-powered
✓  ghc   8.10.3         latest,base-4.14.1.0      hls-powered
✗  ghc   9.0.0.20201227 prerelease,base-4.15.0.0
✗  cabal 2.4.1.0
✗  cabal 3.0.0.0
✔✔ cabal 3.2.0.0        latest,recommended
✗  cabal 3.4.0.0-rc4    prerelease
✔✔ hls   0.8.0          latest,recommended
✔✔ ghcup 0.1.12         latest,recommended
```

Reply:
Sorry for the delay!

Okay.  I see `hls` too. 🤔  Does `ghcup install` install cabal as well as GHC?

Reply:
`ghcup install` can install GHC, cabal, or HLS. I have separate entries for GHC and cabal, but not for HLS, since there is a limit of 8 examples, and hls is one of the minority of use cases.

Reply:
{@navarroaxel, @schneiderl} what would you do here?  Should we only say "GHC", or should we say "GHC/cabal" for this and the other examples?

Reply:
I'd keep it as-is, but I don't know Haskell.